L1Trigger/TrackTrigger/plugins/TTStubBuilder.cc

```diff
@@ -219,9 +219,56 @@ void TTStubBuilder<Ref_Phase2TrackerDigi_>::produce(edm::Event& iEvent, const ed
           // We keep is in the stub final container tough, but flagged as reject by FE
 
           if (FEreject) {
-            tempTTStub2.setTriggerDisplacement(500 + 2. * tempTTStub.getTriggerDisplacement());
-            tempTTStub2.setTriggerOffset(500 + 2. * tempTTStub.getTriggerOffset());
-            tempTTStub2.setRealTriggerOffset(500 + 2. * tempTTStub.getRealTriggerOffset());
+            tempTTStub2.setRawBend(500 + 2. * tempTTStub.getRawBend());
```

Comment:
magic number 500 should be named constant

Reply:
fixed.